wp-content/themes/pub/wporg-learn-2024/patterns/page-my-courses-content.php

```diff
@@ -46,7 +46,7 @@
 
 				<!-- wp:post-excerpt {"showMoreOnNewLine":false,"excerptLength":16,"style":{"typography":{"lineHeight":1.6},"layout":{"selfStretch":"fill","flexSize":null}}} /-->
 
-				<!-- wp:sensei-lms/course-progress {"customBarColor":"var(--wp--custom--color--green-50)","height":8,"className":"wporg-learn-sidebar-course-progress"} /-->
+				<!-- wp:sensei-lms/course-progress {"customBarColor":"var(--wp--custom--color--green-50)","height":8} /-->
```

Comment:
Can you also help remove the same class [here](https://github.com/WordPress/Learn/blob/1152c4bd3dee11f52a65190ff905ab69f8585de6/wp-content/themes/pub/wporg-learn-2024/patterns/sidebar-course.php#L25)? Thanks.

Reply:
Done! https://github.com/WordPress/Learn/pull/2750/commits/360c05b3c7b7ee6a5e9ecddf4e837407feb9b8a9